src/pinning.js

```diff
@@ -308,10 +325,9 @@ class Pinning {
   }
 
   _onMessage (topic, data) {
-    console.log(topic, data)
     if (OrbitDB.isValidAddress(data.odbAddress)) {
       this._sendHasResponse(data.odbAddress)
-      if (data.type === 'PIN_DB') {
+      if (data.type === 'PIN_DB' && this._shouldHandlePinRequest(data)) {
         this.openDB(data.odbAddress, this._openSubStoresAndCacheEntries.bind(this), this._openSubStores.bind(this), null, this.analytics.trackPinDB.bind(this.analytics))
         this.analytics.trackPinDBAddress(data.odbAddress)
       } else if (data.type === 'SYNC_DB' && data.thread) {
```

Comment:
In this conditional we need to check if the given thread is within the whitelisted spaces.